streaming/java/streaming-runtime/src/main/java/org/ray/streaming/runtime/core/resource/ContainerID.java

```diff
@@ -0,0 +1,9 @@
+package org.ray.streaming.runtime.core.resource;
+
+import org.ray.streaming.runtime.core.common.AbstractID;
+
+/**
+ * Container unique identifier.
+ */
+public class ContainerID extends AbstractID {
```

Comment:
May be just use ```AbstractID``` as the type of containerId?

Reply:
The specific implementation class can distinguish different types of ```id```.